src/app/song-widget/song-widget.component.ts

```diff
@@ -26,6 +49,12 @@ export class SongWidgetComponent implements OnInit {
 
   errorMessage: string;
 
+  // animation bools
+  isIn: boolean = true;
+  showTitle: boolean = true;
```

Comment:
Kinda hackey at the moment, may want to revise how this is done for a 1.1 or something